packages/host/tests/integration/components/ask-ai-test.gts

```diff
@@ -0,0 +1,270 @@
+    const mangoId = `${testRealmURL}Pet/mango`;
+    const petCardId = `${testRealmURL}pet.gts`;
+    operatorModeStateService.restore({
+      stacks: [[{ id: mangoId, format: 'isolated' }]],
+      submode: 'code',
+      codePath: petCardId,
+    });
+    setPlaygroundSelections({
+      [`${petCardId.replace('.gts', '')}/Pet`]: {
+        cardId: marcoId,
+        format: 'isolated',
+      },
+    });
+    await renderComponent(
+      class TestDriver extends GlimmerComponent {
+        <template>
+          <OperatorMode @onClose={{noop}} />
+          <CardPrerender />
+        </template>
+      },
+    );
+
+    assert.dom('[data-test-code-mode]').exists();
+    assert.dom('[data-test-ask-ai-input]').hasNoValue();
+    assert.dom('[data-test-ai-assistant-panel]').doesNotExist();
+    await sendAskAiMessage('Change embedded template background to blue');
+    assert
+      .dom('[data-test-ai-assistant-panel] [data-test-chat-title]')
+      .hasText('New AI Assistant Chat');
+    assert
+      .dom('[data-test-pill-menu-header]')
+      .containsText('2 of 2 Skills Active');
+    await assertMessages(assert, [
+      {
+        from: 'testuser',
+        message: 'Change embedded template background to blue',
+        cards: [{ id: marcoId, title: 'Marco' }],
+        files: [{ sourceUrl: petCardId, name: 'pet.gts' }],
+      },
+    ]);
+    await click('[data-test-past-sessions-button]');
+    assert.dom('[data-test-joined-room]').exists({ count: 1 });
+    await click('[data-test-past-sessions-button]');
+
+    // sending new message from AskAI box to open panel
+    await sendAskAiMessage('Change title to Marquitos');
+    await assertMessages(assert, [
+      {
+        from: 'testuser',
+        message: 'Change title to Marquitos',
+        cards: [{ id: marcoId, title: 'Marco' }],
+        files: [{ sourceUrl: petCardId, name: 'pet.gts' }],
+      },
+    ]);
+    assert.dom('[data-test-ask-ai-input]').hasNoValue();
+    assert.dom('[data-test-message-field]').hasNoValue();
+    await click('[data-test-past-sessions-button]');
+    assert.dom('[data-test-joined-room]').exists({ count: 2 });
+    await click('[data-test-past-sessions-button]');
+
+    // sending message to open room via panel's chatbox
+    await fillIn('[data-test-message-field]', 'Goodbye');
+    await click('[data-test-send-message-btn]');
+    await assertMessages(assert, [
+      {
+        from: 'testuser',
+        message: 'Change title to Marquitos',
+        cards: [{ id: marcoId, title: 'Marco' }],
+        files: [{ sourceUrl: petCardId, name: 'pet.gts' }],
+      },
+      {
+        from: 'testuser',
+        message: 'Goodbye',
+        cards: [{ id: marcoId, title: 'Marco' }],
+        files: [{ sourceUrl: petCardId, name: 'pet.gts' }],
+      },
+    ]);
+    await click('[data-test-past-sessions-button]');
+    assert.dom('[data-test-joined-room]').exists({ count: 2 });
+    await click('[data-test-past-sessions-button]');
+  });
```

Comment:
When I tried this in code submode the panel didn’t open:

![screencast 2025-04-02 04-59-19](https://github.com/user-attachments/assets/76d4360a-38db-4178-b872-d4b2ac5e8046)

but maybe this is because I’m using the preview deployment? I see a change in `packages/base/command.gts` that wouldn’t be in that deployment.